app/components/repository-sidebar.js

```diff
@@ -68,20 +70,18 @@ export default Ember.Component.extend({
     return runningAmount + queuedAmount;
   },
 
-  @computed('features.proVersion', 'jobState.runningJobs.[]')
-  runningJobs(proVersion, runningJobs) {
-    if (!proVersion) { return []; }
+  @computed('jobState.runningJobs.[]')
+  runningJobs(runningJobs) {
     return runningJobs;
   },
 
-  @computed('features.proVersion')
-  queuedJobs(proVersion) {
-    if (!proVersion) { return []; }
-
+  @computed('currentUser')
+  queuedJobs(user) {
     const queuedStates = ['created'];
     const result = this.get('store').filter(
       'job',
-      job => queuedStates.includes(job.get('state'))
+      job =>
+        queuedStates.includes(job.get('state') && user && user.hasAccessToRepo(job.get('repo')))
```

Comment:
`isCurrentUserACollaborator` is sort of a [hack that I recently added](https://github.com/travis-ci/travis-web/blob/d285d2f5a5de897f017e000054648473d52db75f/app/models/repo.js#L36-L43) just to make permissions check a computed property. In the future we will get it from the API as `@is_collaborator` property.